fuse_publishers/src/pose_2d_publisher.cpp

```diff
@@ -199,7 +200,7 @@ void Pose2DPublisher::onStop()
   // Stop the tf timer
   if (publish_to_tf_)
   {
-    tf_publish_timer_.stop();
+    tf_publish_timer_.reset();
```

Comment:
I think `stop()` in ROS 1 translates to `cancel` in ROS 2 https://github.com/ros2/rclcpp/blob/7c6785176a878cf16eb98995b732f405b9e776ed/rclcpp/include/rclcpp/timer.hpp#L74

Reply:
https://github.com/locusrobotics/fuse/pull/283/commits/59d10ddc448f2a21928d5104dabe494e0e37f8e4